tests/kuttl/common/assert-sample-deployment.yaml

```diff
@@ -59,9 +59,6 @@ spec:
   serviceUser: "heat"
 status:
   databaseHostname: openstack
-  heatAPIReadyCount: 1
```

Comment:
I think it makes sense to keep heatApiReadyCount . We can also assert the following two fields.

heatEngineReadyCount: 1
transportURLSecret: rabbitmq-transport-url-heat-heat-transport

Reply:
You're right, before for some reason these fields were not exposed, but now they appear, I'll put them back